packages/expo-analytics-amplitude/src/ExpoAmplitude.web.ts

```diff
@@ -1,5 +1,21 @@
+import amplitude from 'amplitude-js'
+
 export default {
   get name(): string {
     return 'ExpoAmplitude';
   },
-};
+  initialize: (apiKey: string, options?: amplitude.Config) => {
+    amplitude.getInstance().init(apiKey, undefined, options)
+  },
+  setUserId: amplitude.getInstance().setUserId,
```

Comment:
This is the format we use in all of the web modules.
```suggestion
  setUserId( ... ) {
```